circuits/ecsda-circom/test/eth_addr.ts

```diff
@@ -5,6 +5,8 @@ import { setupDirectories } from "../../test_utils"
 const pathToCircom = "./eth_addr.circom"
 
 describe("Test eth_addr", function () {
+    this.timeout(100000);
```

Comment:
I already added the timeout in here `.mocharc.js` so you don't need to add it again